### ISSUE TITLE
Arabic Slugs in translated Routes are being stuck

### ISSUE THREAD

**Issue by legend6** [mcamara/laravel-localization#758]
I have the same problem as our friend @mkwsra in error number #174, but using arabic slugs, static translations work fine, but I only have this problem

i.e:

xxx.com/en/posts/title

works just fine, chaning it into:
xxx.com/ar/posts/العنوان

works fine but:

when I try to change language into English :
xxx.com/en/posts/العنوان

my resources/lang/ar/routes.php file
`'posts.show' => 'posts/{post}',`

resources/lang/en/routes.php
`'posts.show' => 'posts/{post}',`

my web.php
```
Route::prefix(LaravelLocalization::setLocale())
    ->middleware(['localizationRedirect', 'localeSessionRedirect', 'localeViewPath', 'localize'])
    ->group(function(){
        Route::get(LaravelLocalization::transRoute('routes.posts.show'), [PostController::class, 'show'])->name('posts.show');
}
```

my Post model
```
public function getLocalizedRouteKey($locale)
{
    return $this->getTranslation('slug', $locale);
}
```

**More info:**
- Version of Laravel: v8.13
- Version of the Laravel-localization package: v1.6.1

**Additional context**

**Comment by mkwsra** [comment]
For me, I've been using this library instead for a long time now [https://github.com/ARCANEDEV/Localization](https://github.com/ARCANEDEV/Localization) if you wish give it a try.

**Comment by legend6** [comment]
> 
> 
> For me, I've been using this library instead for a long time now https://github.com/ARCANEDEV/Localization if you wish give it a try.

thank you @mkwsra for your answer, and I also want to thank you for your wonderful laravel-acquaintances package, I have already tested it but I prefer laravel-localization by mcamra, I did not encounter any problem except that, and i hope @mcamara will fix it ;)

**Comment by legend6** [comment]
Problem fixed in this pull request #722, thanks to @djemmal-nour-el-islam ;)
